package.json

```diff
@@ -1,7 +1,10 @@
 {
+  "name": "cardanow-ts",
+  "version": "0.0.0.1",
```

Comment:
@aciceri I think this is the duplication I was talking about.

For some reason it looks we have to put in both the nix and in the json file so I will use importJSON to have a single source of truth